patches/components-permissions-BUILD.gn.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/components/permissions/BUILD.gn b/components/permissions/BUILD.gn
+index 799755985b575a7464897d0969bf383c386d267c..736acd4686d8cc47f09b5859b9ef78644c73d3c1 100644
+--- a/components/permissions/BUILD.gn
++++ b/components/permissions/BUILD.gn
+@@ -2,6 +2,8 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//brave/components/permissions/sources.gni")
```

Comment:
this should be done on the same line as sources += cc @mkarolin

Reply:
fixed here https://github.com/brave/brave-core/pull/8459